jdaviz/configs/imviz/plugins/footprints/footprints.py

```diff
@@ -79,6 +83,7 @@ class Footprints(PluginTemplateMixin, ViewerSelectMixin):
     fill_opacity = FloatHandleEmpty(0.1).tag(sync=True)
 
     # PRESET OVERLAYS AND OPTIONS
+    has_pysiaf = Bool(preset_regions._has_pysiaf).tag(sync=True)

```

Comment:
Why does this now have to be a traitlet with sync? It should be static value at import check time, right?

Reply:
because I rely on it to show a warning in the UI (maybe sync isn't necessary, but definitely needs to be a traitlet)